services/gitlab/gitlab-coverage.tester.js

```diff
@@ -3,59 +3,63 @@ import { createServiceTester } from '../tester.js'
 export const t = await createServiceTester()
```

Comment:
Could you add a couple of test cases in here for the redirector: one with a branch and one without. There is a test helper `.expectRedirect()` you can use to make sure the correct redirect was issued.

Reply:
I renamed `gitlab-coverage-service` to `gitlab-pipeline-coverage.service`
to reflect the change in the URL path.

I added `gitlab-coverage-redirect.service`. In this second I placed tests with redirector (`branch`+`job_name`/`gitlab_url`).

Since `branch` was a required param in the previous `gitlab coverage` route, we cannot add a test without it to test the redirection ;)
![image](https://user-images.githubusercontent.com/31891675/192786894-8ea5715c-67bb-4d79-970a-db2b09b3af4f.png)